handlers/fulfilment-date-calculator/src/main/scala/com/gu/supporter/fulfilment/GuardianWeekly.scala

```diff
@@ -29,7 +29,8 @@ object GuardianWeeklyFulfilmentDates extends FulfilmentConstants(
         deliveryAddressChangeEffectiveDate(today),
         holidayStopFirstAvailableDate(today),
         holidayStopProcessorTargetDate(today),
-        finalFulfilmentFileGenerationDate(today)
+        finalFulfilmentFileGenerationDate(today),
+        deliveryAddressChangeEffectiveDate(today)
```

Comment:
if the function name doesn't align to what's being set, could we please use a named param
```suggestion
        newSubscriptionEarliestStartDate = deliveryAddressChangeEffectiveDate(today)
```